assets/src/edit-story/karma/fixture/containers/designPanel/filter.js

```diff
@@ -30,34 +29,18 @@ export class Filter extends AbstractPanel {
   }
 
   get none() {
-    return this._get(
-      this.getByRole('checkbox', { name: /Set filter: None/i }),
-      'none',
-      Toggle
-    );
+    return this.getByRole('button', { name: /Set filter: None/i });
```

Comment:
I actually think we should update these labels now that we're at it. Probably to something like: "No filter", "Tint filter", "Linear filter", "Vignette filter" or something like that. The verb-version here is misunderstood ARIA.